inst/rmarkdown/templates/arxiv/resources/template.tex

```diff
@@ -66,6 +66,13 @@
 \newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
 $endif$
 
+% Pandoc header
+$for(header-includes)$
+$header-includes$
+$endfor$
+
+$preamble$
+
```

Comment:
Why add a preamble variable ? 
`header-includes` variable is used for that already. Using `includes` argument in the yaml header for example.
https://bookdown.org/yihui/rmarkdown/pdf-document.html#includes-1

Reply:
This makes it consistent with other templates.

Also people expect for `preamble` to work: https://github.com/rstudio/rticles/issues/257#issuecomment-564304252

Reply:
Oh I see. so this is a specificity of some **rticles** templates. Not all have it, and it won't work with **rmarkdown**, so it could be expected but by some **rticles** user only. 

Fine by me. Thank you for pointing this out!